src/uct/ib/mlx5/gdaki/gdaki.c

```diff
@@ -14,8 +14,15 @@
 #include <ucs/datastruct/string_buffer.h>
 #include <uct/ib/mlx5/rc/rc_mlx5.h>
 #include <uct/cuda/base/cuda_iface.h>
+#include <uct/cuda/cuda_copy/cuda_copy_md.h>
 
 #include <doca_log.h>
+#include <cuda_runtime.h>
+#include <cuda.h>
+
+
+#define UCT_GDAKI_DOCA_NOTUSE    1
+#define UCT_GDAKI_DOCA_NOTUSEPTR (void*)1
```

Comment:
why is it (void*)1 and not NULL / 0 ?
Is it a DOCA API constant?

Reply:
DOCA checking that all arguments are not null, otherwise return DOCA_ERROR_INVALID_VALUE